content/en/docs/Contribute/style-guide.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Documentation style guide
+linkTitle: Style guide
+weight: 40
+slug: style-guide open-telemetry
+---
+
+The OpenTelemetry documentation adheres to the
+[Kubernetes Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/)
+and the
+[Google Developer Documentation Style Guide](https://developers.google.com/style)
+. The following sections contain guidance that is specific to the OpenTelemetry
+project.
+
+## OpenTelemetry.io word list
+
+A list of OpenTelemetry-specific terms and words to be used consistently across
+the site.
+
+<!-- prettier-ignore-start -->
+| Term | Usage |
+| --- | --- |
+| OpenTelemetry | OpenTelemetry should always be capitalized. Don't use Open-Telemetry. |
+| OTel | OTel is the accepted short form of OpenTelemetry. Don't use OTEL. |
+| Collector | When referring to the OpenTelemetry Collector, always capitalize Collector. |
+| Repository | Code repositories. Don't use repo or repos. |
```

Comment:
nice!

Also add: OTEP, OpAMP, OTLP

Repository can also be repository, this might be confusing from that list

Reply:
Added!